go.mod

```diff
@@ -1,17 +1,19 @@
 module github.com/segmentio/chamber/v3
 
-go 1.20
+go 1.21
+
+toolchain go1.21.6
 
 require (
 	github.com/alessio/shellescape v1.4.2
-	github.com/aws/aws-sdk-go-v2 v1.30.3
-	github.com/aws/aws-sdk-go-v2/config v1.27.27
-	github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.16.11
+	github.com/aws/aws-sdk-go-v2 v1.30.4
+	github.com/aws/aws-sdk-go-v2/config v1.27.28
+	github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.16.12
 	github.com/aws/aws-sdk-go-v2/service/s3 v1.57.1
 	github.com/aws/aws-sdk-go-v2/service/secretsmanager v1.32.1
```

Comment:
Post merge of https://github.com/segmentio/chamber/pull/559, https://github.com/segmentio/chamber/pull/560 was opened which also shows bumps to these two packages. Should we bump these as well to keep everything aligned?

Reply:
Oh yesh, adding them